offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/CreateOfferViewModel.groovy

```diff
@@ -151,9 +151,9 @@ class CreateOfferViewModel {
         offerCreatedSuccessfully = false
 
         offerId = null
-        projectTitle = ""
-        projectObjective = ""
-        experimentalDesign = ""
+        projectTitle = null
+        projectObjective = null
+        experimentalDesign = null
```

Comment:
This is optional and the controller knows that. It should not throw a null pointer if it is indeed still null in the controller.